packages/docs-reanimated/src/examples/ReduceMotionConfig.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import { StyleSheet, View } from 'react-native';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withRepeat,
+  withTiming,
+  ReduceMotionConfig,
+  ReduceMotion,
+} from 'react-native-reanimated';
+
+export default function App() {
+  const sv = useSharedValue<number>(0);
+  const animatedStyle = useAnimatedStyle(() => ({
+    transform: [{ rotate: `${sv.value}deg` }],
+  }));
+
+  useEffect(() => {
+    sv.value = withRepeat(withTiming(360, { duration: 2000 }), -1, true);
+  });
+
+  return (
+    <View style={styles.container}>
+      {/* I'll uncomment if after merge of ReduceMotionConfig to main and new nightly release, because now component is unavailable */}
+      {/* <ReduceMotionConfig mode={ReduceMotion.Never} /> */}
+      <Animated.View style={[styles.box, animatedStyle]} />
+    </View>
```

Comment:
If possible, I'd add some simple reduce motion mode switch to the interactive example. Some less Reanimated-experienced docs users won't be able to tell if the animation worked before applying the `.Never` mode or not

Reply:
make sense, I'll do it 🫡